token-syncer/bin/ci/run-integration-tests.sh

```diff
@@ -16,7 +16,7 @@ TEST_SELECTOR=${2:-integration}
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 PROJECT_DIR=${DIR}/../../../
 WAITER_DIR=${PROJECT_DIR}/waiter
-KITCHEN_DIR=${PROJECT_DIR}/test-apps/kitchen
+KITCHEN_DIR=${PROJECT_DIR}/containers/test-apps/kitchen
```

Comment:
I missed updating this path on my first pass after reorganizing the directory structure, but no tests failed. Do we even need this?

Reply:
I'm surprised that it worked, the `test-ping-tokens` should use it.

Reply:
I checked the logs and the tests were skipped. Mystery solved.